scripts/split-e2e-tests.ts

```diff
@@ -35,7 +35,7 @@ const WINDOWS_TEST_ALLOWLIST: string[] = [
   'schema-auth-11-c_pkg',
   'auth_6_pkg',
   'frontend_config_drift_pkg',
-  'hooks_pkg',
+  'hooks-b_pkg',
```

Comment:
where is hooks-a?

Reply:
good catch, should include both a/b here

Reply:
This windows allow list , hooks_a file wont run on windows, only on linux

Reply:
ok then I'll resolve